.zenodo.json

```diff
@@ -136,11 +139,17 @@
       "name": "Thirion, Bertrand",
       "orcid": "0000-0001-5018-7895"
     },
+    {
+      "name": "Chadwick Boulay"
```

Comment:
@cboulay Do you want to be an author on the Zenodo, and if so, what affiliation, name and [ORCID](http://orcid.org/) should we use?

Reply:
Sure, thanks!
Ottawa Hospital Research Institute
0000-0003-1747-3931